### ISSUE TITLE
Can we reduce requirements of libzip?

### ISSUE THREAD

**Issue by dioni21** [tihmstar/libfragmentzip#10]
At files `libfragmentzip.pc.in` and `configure.ac`, there's a specification requiring `libzip >= 1.0`

This makes impossible to compile for CentOS 7.

Changing these to `libzip >= 0.10` makes it compile.

Do we really need 1.0?

**Comment by tihmstar** [comment]
Have you tried updating your system? o.O

**Comment by dioni21** [comment]
I will, when RedHat release RHEL8.   ;-)

The big question is: "Do we really need 1.0?".  I could compile without errors, but did not test execution.

**Comment by tihmstar** [comment]
TLDR: I will not downgrade the minimum required version, but feel free to edit your local clone to accept an older version.



So my thoughts on this are:
People should keep their systems up to date, this is super important especially with security bugs/fixes in mind.
When i start a project i use the latest available version of a library exactly for this reason and code, compile and link with that version.

If you really really can't get the most recent version or for some reason just *have to* use an outdated version, i would suggest simply trying to manually patch the ```configure.ac``` and see if it still works correctly (which very well might be the case). 
However if you do that, please do not expect any kind of support.
